DDSim/DDSim/DD4hepSimulation.py

```diff
@@ -89,6 +89,8 @@ def __init__(self):
     self.filter = Filter()
     self.physics = Physics()
 
+    self.argv = None
```

Comment:
use 
```
self._argv
```
because, for example, we don't need this to show up the steering file when dumped